### PR TITLE
Marine Struct Stat Edits

### PR DESCRIPTION
**Pull request by OH296**
- edits from pr #76 
- should all be in line with review

### CODE REVIEW

File: scripts/scr_apothecary_ship/scr_apothecary_ship.gml [hunk 1]
```diff
@@ -68,36 +68,14 @@ function scr_apothecary_ship() {
 	// show_message("[1] "+string(obj_ini.ship[1])+": "+string(ship_apoth[1])+" apothecaries");
 
 
-	var normal_hp,mixhp,ratio;
+	var normal_hp,mixhp,ratio,unit, unit_location;
```
Comment (GalacticChimp) [inline scripts/scr_apothecary_ship/scr_apothecary_ship.gml:71]: `unit_location` is not used at all?
`unit` can be deleted from here, it's declared and used about 30 lines below

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 2]
```diff
@@ -1,1352 +1,1338 @@
...
-	        gladius=7;
-	        hunters=3;
-		
-	        if (global.chapter_name="Imperial Fists")
-	           {
-				   flagship_name="Spear of Vengeance";
-				   battle_barges+=1;
-			   }
-	    
-		if (global.chapter_name="Crimson Fists")
```
Comment (GalacticChimp) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:217]: why is this deleted? it's for the new Chapter

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 3]
```diff
@@ -1,1352 +1,1338 @@
...
-	if (global.chapter_name="Dark Angels"){veteran=0;terminator+=80;}
-	if (global.chapter_name="Lamenters"){
-	    tenth=0;ninth=0;eighth=0;seventh=0;sixth=0;fifth=0;
-	    techs=8;epistolary=6;apothecary=5;codiciery=3;lexicanum=5;terminator=5;veteran+=10;
-	}
-	if (global.chapter_name="Soul Drinkers"){
-		tenth-=38;seventh=0;sixth=40; assault-=10;
-		fifth-=20;fourth-=20;third-=20;second-=20;terminator-=5;veteran-=20;
-	}
-if (global.chapter_name="Crimson Fists"){
```
Comment (GalacticChimp) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:442]: same as above

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 4]
```diff
@@ -1,1352 +1,1338 @@
...
+	    role[i,14]="Chaplain";wep1[i,14]="Power Sword";wep2[i,14]="Storm Bolter";armour[i,14]="Power Armour";gear[i,14]="Rosarius";mobi[i,14]="";
+	    role[i,15]="Apothecary";wep1[i,15]="Power Sword";wep2[i,15]="Storm Bolter";armour[i,15]="Power Armour";gear[i,15]="Narthecium";mobi[i,15]="";
+	    role[i,16]="Techmarine";wep1[i,16]="Power Axe";wep2[i,16]="Storm Bolter";armour[i,16]="Power Armour";gear[i,16]="Servo Arms";mobi[i,16]="";
+	    role[i,17]="Librarian";wep1[i,17]="Force Weapon";wep2[i,17]="Storm Bolter";armour[i,17]="Power Armour";gear[i,17]="Psychic Hood";mobi[i,17]="";
+	}// 100 is defaults, 101 is the allowable starting equipment // info
+		//made all the exp buffs sort into neat little structs so theyre easier to dev and player modify
+		/*each position in array the represents a company (with position 0 representing specials), and the first position in each second dimension
+			is the experience bonus each marine will get in that company modified with the gauss function where the second position is one standard deviation*/
+	company_spawn_buffs = [0,0,[110,15],[105,15],[95,15],[80,10],[65,10],[55,10],[45,10],[35,10],[3,10]]
```
Comment (GalacticChimp) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:527]: +1

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
please check my comments

File: scripts/scr_apothecary_ship/scr_apothecary_ship.gml (hunk 1 above)
Comment (OH296) [reply scripts/scr_apothecary_ship/scr_apothecary_ship.gml:71]: just declared at top of page as otherwise it's just repeated declaration

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 2 above)
Comment (OH296) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:217]: something must have gone wrong when i pulled the branch back to work on it. fixed.

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 3 above)
Comment (OH296) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:442]: same as above.fixed

**Comment by OH296** [comment]
all points addressed

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 2 above)
Comment (GalacticChimp) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:217]: this is still deleted

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 3 above)
Comment (GalacticChimp) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:442]: this is still deleted

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
some comments waiting to be resolved

**Comment by GalacticChimp** [comment]
what do skulls mean?

image

File: scripts/scr_apothecary_ship/scr_apothecary_ship.gml (hunk 1 above)
Comment (GalacticChimp) [reply scripts/scr_apothecary_ship/scr_apothecary_ship.gml:71]: if it's not used, why is it there?

File: scripts/scr_initialize_custom/scr_initialize_custom.gml (hunk 3 above)
Comment (OH296) [reply scripts/scr_initialize_custom/scr_initialize_custom.gml:442]: oh shit yeah sorry i made a comment here then had to run out into the yard before i physically made the push and was then tied up all day

**Comment by OH296** [comment]
sculls means the marine is a boarder during fleet combat

**Comment by GalacticChimp** [comment]
game breaks when you start as Crimson Fists (worked previously)

image

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
game breaks when starting as Crimson Fists (worked previously)

image

**Review by GalacticChimp — APPROVED** [review]

**Review by GalacticChimp — APPROVED** [review]
